src/Components/JournalPrompt/JournalPrompt.js

```diff
@@ -5,21 +5,12 @@ import './JournalPrompt.css';
 const JournalPrompt = ( {updateJournal} ) => {
   const [newEntry, setNewEntry] = useState('');
 
-  const updateEntry = () => {
-    const input = document.getElementById('journal-entry');
-    setNewEntry(input.value);
-  }
-
-  const submitEntry = () => {
-    updateJournal(newEntry);
-  }
-
   return (
     <div className="prompt-container">
       <form>
         <h2 className="prompt">Why are you feeling this way?</h2>
-        <textarea id="journal-entry" onChange={updateEntry}></textarea>
-        <Link to="/how-you-felt" className="uni-btn" onClick={submitEntry}>→</Link>
+        <textarea id="journal-entry" onChange={(event) => setNewEntry(event.target.value)}></textarea>
+        <Link to= "/how-you-felt" className="uni-btn" onClick={() => updateJournal(newEntry)}>→</Link>
```

Comment:
Chef's Kiss KB - LOVE to see you learning and implementing concepts simultaneously! This looks super clean and is much easier to follow. Great work :)